components/UI/Copyright/Copyright.module.css

```diff
@@ -1,16 +1,19 @@
 .copyright {
     position: fixed;
     z-index: 1000;
-    font-size: .5vw;
+    font-size: .6vw;
     color: rgba(255,255,255,.8);
     right: 0;
     bottom: 1px;
     background: rgba(0,0,0,.5);
     margin-right: .25vw;
 }
 
-@media screen and (max-width: 768px) {
+/* kinda MAX_MOBILE_WIDTH constant but media-queries can't work with css-variables :( */
```

Comment:
Yes, you're right - it's very sad :(

In fact, it will appear pretty soon. It is also supported in `postcss-preset-env`, but Next.js use `stage: 2`, not `3`.

https://preset-env.cssdb.org/features/#custom-media-queries
https://www.w3.org/TR/mediaqueries-5/#custom-mq

Reply:
Конфигурация через postcss.config.json, к сожалению, требует переустановки всех зависимостей

https://nextjs.org/docs/pages/building-your-application/configuring/post-css#customizing-plugins

Reply:
It is also supported in `postcss-preset-env`, but Next.js use `stage: 2`, not `3`.

Reconfiguration via `postcss.config.json` unfortunately requires reinstalling all dependencies (postcss-presset-env, autoprefixier, postcss-flexbugs-fixes)

https://nextjs.org/docs/pages/building-your-application/configuring/post-css#customizing-plugins